float64 motor_current 
float64 motor_rpm   
int32 motor_pulses   
float64 motor_temperature   

float64 driver_voltage   
float64 driver_temperature   
int32 driver_state   